### ISSUE TITLE
[REQUEST] JustFlight Fokker F28

### ISSUE THREAD

**Issue by viert** [Sequal32/yourcontrols#224]
Would be really nice to have

**Comment by viert** [comment]
Not sure how to remove the "bug" label, it showed up automatically and seems to be read-only for me

**Comment by rpgamerous** [comment]
Yes! I know the first post here is a few months old, but I'd like to add my weight to this request. One of the MSFS planes that's a joy to fly but would still greatly benefit (and maybe even be MORE fun) from the shared cockpit experience!
